merlin/loader/ops/embeddings/torch_embedding_op.py

```diff
@@ -0,0 +1,248 @@
+import numpy as np
+import torch
+from torch.nn import Embedding
+
+from merlin.core.protocols import Transformable
+from merlin.dag import BaseOperator
+from merlin.dag.selector import ColumnSelector
+from merlin.schema import ColumnSchema, Schema, Tags
+
+
+class TorchEmbeddingOperator(BaseOperator):
```

Comment:
Same as in tensorflow case, so many of the operators are just a little different, but to avoid confusions and allow users to understand more clearly uses and use cases we have kept these operators separate. Would be good to move to a state where we just have one operator for this (as previously stated).